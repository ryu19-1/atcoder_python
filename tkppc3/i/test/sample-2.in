10 3
1 2 3 3 1 6 7 8 8
2 3 5 3 1 1 6 2 7
1 3 2 2 3 2 3 1 1 3
