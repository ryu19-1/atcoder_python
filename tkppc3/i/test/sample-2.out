3
5
8
18
11
2
3
13
17
21
